projects/packages/forms/src/blocks/contact-form/components/jetpack-field.js

```diff
@@ -51,8 +50,9 @@ const JetpackField = props => {
 					className="jetpack-field__input"
 					onChange={ e => setAttributes( { placeholder: e.target.value } ) }
 					style={ fieldStyle }
-					type={ type }
-					value={ placeholder }
+					type={ isSelected ? 'text' : type }
```

Comment:
This is so that you can enter text placeholders for number input. Otherwise, you could only enter numbers.

The downside is that you might lose `input[type="number"]` specific styling:

<img width="634" alt="Screenshot 2025-02-27 at 17 02 30" src="https://github.com/user-attachments/assets/28184ebc-7d55-4bca-b21c-81509fc740c8" />